addons/cluster-autoscaler/README.md

```diff
@@ -4,6 +4,10 @@ Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernet
 - there are pods that failed to run in the cluster due to insufficient resources.
 - there are nodes in the cluster that have been underutilized for an extended period of time and their pods can be placed on other existing nodes.
 
+## Prerequisites
+Auto-discovery finds ASGs tags as below and automatically manages them based on the min and max size specified in the ASG.
+- Tag the ASGs with keys to match .Values.autoDiscovery.tags, by default: `k8s.io/cluster-autoscaler/enabled` and `k8s.io/cluster-autoscaler/<YOUR CLUSTER NAME>`
```

Comment:
please add the `value` part of above **key** tags

Reply:
I added value for tag key as you mentioned.